packages/dev-server/src/dev-server.ts

```diff
@@ -1,29 +1,20 @@
 import type http from 'http'
 import { getRequestListener } from '@hono/node-server'
-import type { Miniflare, MiniflareOptions, WorkerOptions } from 'miniflare'
 import type { Plugin, ViteDevServer, Connect } from 'vite'
+import { getEnv as cloudflarePagesGetEnv } from './cloudflare-pages/index.js'
+import type { Env, Fetch, EnvFunc } from './types.js'
 
 export type DevServerOptions = {
   entry?: string
   injectClientScript?: boolean
   exclude?: (string | RegExp)[]
-  cf?: Partial<
-    Omit<
-      WorkerOptions,
-      // We can ignore these properties:
-      'name' | 'script' | 'scriptPath' | 'modules' | 'modulesRoot' | 'modulesRules'
-    > &
-      Pick<
-        MiniflareOptions,
-        'cachePersist' | 'd1Persist' | 'durableObjectsPersist' | 'kvPersist' | 'r2Persist'
-      > & {
-        // Enable `env.ASSETS.fetch()` function for Cloudflare Pages.
-        assets?: boolean
-      }
-  >
+  env?: Env | EnvFunc
+} & {
+  // The `cf` option is for backward compatibility. Typically, you should use `env`.
```

Comment:
Is this better to add `@deprecate` JSDoc?

Reply:
Yeah. I've added.